### PR TITLE
Sample code of using streamcut with Hadoop connector

### PR DESCRIPTION
**Pull request by yangb8**
Signed-off-by: Bo Yang <b.yang@emc.com>

Same change as PR #137, but the target branch is 'develop' but not 'master'.

### CODE REVIEW

**Comment by RaulGracia** [comment]
Please @yangb8, consider either modifying PR #137 to target `develop` and close this PR or close PR #137 and keep this one.

**Comment by yangb8** [comment]
PR #137 has been closed.

File: hadoop-connector-examples/README.md [hunk 1]
```diff
@@ -38,15 +38,40 @@ Hadoop (verified with Hadoop 2.8.3 on Ubuntu 16.04)
...
    hadoop fs -rmr ${HADOOP_EXAMPLES_INPUT_DUMMY}
-   hadoop fs -rmr ${HADOOP_EXAMPLES_OUTPUT}
 
-4. generate words into pravega
-   hadoop jar ${HADOOP_EXAMPLES_JAR} randomtextwriter -D mapreduce.randomtextwriter.totalbytes=32000 ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
+4. run hadoop commands
 
-5. run hadoop command
-   hadoop jar ${HADOOP_EXAMPLES_JAR} ${CMD} ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM} ${HADOOP_EXAMPLES_OUTPUT}
+   4.1 to process all events in the stream
```
Comment (RaulGracia) [inline hadoop-connector-examples/README.md]: I tested the example and it works. Maybe the way of instantiating stream cuts is not the most intuitive one from a user's perspective. I would suggest to clarify a little bit more in the `README` file what is the user achieving executing these steps and at least add a reference to the concept of stream cut (see http://pravega.io/docs/latest/streamcuts/).

File: hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java [hunk 2]
```diff
@@ -111,15 +111,29 @@ public static void main(String[] args) throws Exception {
         Configuration conf = new Configuration();
         String[] otherArgs = new GenericOptionsParser(conf, args).getRemainingArgs();
         if (otherArgs.length < 5) {
-            System.err.println("Usage: wordcount <dummy_hdfs> <uri> <scope> <stream> <out>");
+            System.err.println("Usage: wordcount <dummy_hdfs> <uri> <scope> <stream> <out> <optional start positions> <optional end positions>");
```
Comment (RaulGracia) [inline hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java]: Maybe this is more specific: `<optional start stream cut>` / `<optional end stream cut>`

File: hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java [hunk 3]
```diff
@@ -131,11 +145,14 @@ public static void main(String[] args) throws Exception {
...
 
         FileInputFormat.addInputPath(job, new Path(otherArgs[0]));
-        Path outputpath = new Path(otherArgs[4]);
+        Path outputpath = new Path(out);
         FileOutputFormat.setOutputPath(job, outputpath);
 
         boolean result = job.waitForCompletion(true);
         readAndPrint(outputpath, conf);
+
+        System.out.printf("End positions of stream cut %s/%s: '%s'\n", scope, stream, endPos);
```
Comment (RaulGracia) [inline hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java]: Consider use logging for these purposes.

File: hadoop-connector-examples/README.md (hunk 1 above)
Comment (yangb8) [reply hadoop-connector-examples/README.md]: accepted. added this link

File: hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java (hunk 2 above)
Comment (yangb8) [reply hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java]: accepted.

File: hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java (hunk 3 above)
Comment (yangb8) [reply hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java]: accepted.

**Comment by vijikarthi** [comment]
can we merge this code?

**Pull request merged**